packages/edit-site/src/components/sidebar-navigation-screen-library/index.js

```diff
@@ -114,7 +117,38 @@ export default function SidebarNavigationScreenLibrary() {
 										<CategoryItem
 											key={ category.name }
 											count={ category.count }
-											label={ category.label }
+											label={
+												<Flex
+													justify="left"
+													align="center"
+													gap={ 0 }
+												>
+													{ category.label }
+													<Tooltip
+														position="top center"
+														text={ __(
+															'Theme patterns cannot be edited here'
+														) }
+													>
+														<span
+															style={ {
+																display:
+																	'inline-flex',
+															} }
```

Comment:
We already have stylesheets for the `PageLibrary` and the `SidebarNavigationScreenLibrary` components, can we leverage those and avoid the inline styles?

Reply:
Yeah, pretty sure we can. Just wasn't sure what was the best for this one-shot style. Like below we have the same inline style for SVG icon, which I copied from somewhere else.